ipatests/test_integration/test_installation.py

```diff
@@ -196,3 +197,36 @@ def test_install_master(self):
 
     def test_install_kra(self):
         tasks.install_kra(self.master, first_instance=True)
+
+
+@pytest.mark.ds_acceptance
+class TestDSAcceptance(IntegrationTest):
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=True)
+
+    def test_install_replica0(self):
+        tasks.install_replica(self.master, self.replicas[0], setup_ca=False,
+                              setup_kra=False, setup_dns=False)
+
+
+@pytest.mark.cs_acceptance
+class TestCSAcceptance(IntegrationTest):
```

Comment:
Are you sure that that marking TestInstallWithCA_KRA_DNS1 would result in:
- installation of master with dns, ca
- installation of replica with ca, no dns
-  installation of kra on the master 
-  instalation of kra on the second replica
as was agreed some time ago?